docs/en/quick-answers/troubleshooting/cant-use-file-linking-toolkit-app-firefox.md

```diff
@@ -41,7 +41,7 @@ You need to add an exception to the Firefox browser so that the certificate is a
 
 Click **Get Certificate** and then **Confirm Security Exception**. You can ignore the message about the certificate not being trusted. This is normal as the certificate was generated by Shotgun Desktop and not by a certification authority.
 
-You should now be able to access local file linking and launch Toolkit applications from your Shotgun site. If you are still experiencing issues after adding the exception, please contact our [support team](support@shotgunsoftware.com).
+You should now be able to access local file linking and launch Toolkit applications from your Shotgun site. If you are still experiencing issues after adding the exception, please contact at support@shotgunsoftware.com.
```

Comment:
there's a word missing here: "please contact **us** at" or something like that (or just get rid of "at")